README.md

```diff
@@ -275,6 +275,8 @@ Options:
   - `patch = true` Whether we want to use the `PATCH` verb and semantics, sending
   only the changed attributes instead of the whole resource down the wire.
   - `keepChanges = false` Whether we want to keep the changes after the response from the API.
+  - `endpoint` (Optional) Target API endpoint where we want perform the save action. It
```

Comment:
I find this parameter name misleading... Wouldn't be beter to call it `path` since it's the relative last part of the url?

Reply:
Yep, I'm was thinking about `endpoint`, `url` or `path`. Maybe `path` is the best one. I don't wanted to use `url` because collides with the model `url` function and I didn't want to mix things.

Reply:
path 👍